Author: <gaudenz@debian.org>
Description: send error message to stderr if no uinput device is found
--- a/mouseemu.c
+++ b/mouseemu.c
@@ -821,8 +821,8 @@
 
 	running = uinput_setup();
 	if (running < 0) {
-		syslog(LOG_NOTICE, "Make sure uinput module is loaded or available "
-		                   "in the kernel.\n");
+		fprintf(stderr, "No uinput device found! Make sure the uinput module is loaded\n"
+		                "or CONFIG_INPUT_UINPUT is compiled into the kernel.\n");
 	}
                                                          
 
